package.json

```diff
@@ -70,7 +70,8 @@
 		"php:lint": "composer phpcs:lint",
 		"reformat-files": "prettier --ignore-path .eslintignore --write \"**/*.{js,jsx,cjs,ts,json}\"",
 		"validate-es5": "pnpx eslint --parser-options=ecmaVersion:5 --no-eslintrc --no-ignore",
-		"version-packages": "bash ./tools/version-packages.sh"
+		"version-packages": "bash ./tools/version-packages.sh",
+		"storybook:build": "cd projects/js-packages/storybook && pnpm run storybook:build"
```

Comment:
Instead of this, we've had project build commands in the project's `composer.json`. We can do that and be able to run it with `jetpack build js-packages/storybook`

Reply:
932af7aee336a8cfa8fce0dec23e76e822673c7e